specs/activity-api.yaml

```diff
@@ -202,48 +173,35 @@ paths:
             type: string
         - name: commandIndex
           description: >
-            Wait until command with the specified index finishes. Must be accompanied by a valid "timeout" query parameter.
+            Wait until command with the specified index finishes. Must be accompanied by a valid "pollTimeout" query parameter.
```

Comment:
@mfranciszkiewicz you are removing `timeout` in this PR, but the comment is unchanged and still requires to specify `timeout`